docs/usage/applications/service-access/_index.md

```diff
@@ -0,0 +1,404 @@
+---
+title: Access service from outside Kubernetes cluster
```

Comment:
Why do you think this guide belongs to g/g?
And in this PR I have already seen other guide for accessing services via `k port-forward` and ingress, probably you will want to merge them together.